std/conv.d

```diff
@@ -3737,10 +3737,21 @@ unittest
 @property T octal(T, string num)()
     if (isOctalLiteral!num)
 {
+    return octal!T(num);
+}
+
+/// Ditto
+@property T octal(T)(const(char)[] num) pure nothrow
+    if (isNumeric!T)
+{
+    assert( isOctalLiteralString(num),
```

Comment:
This precondition must be documented. The result will be undefined when `num` contains non-octal-digit-characters.